modules/dhcp_common/ping.rb

```diff
@@ -0,0 +1,58 @@
+require 'timeout'
+require 'socket'
+
+module Proxy::DHCP
```

Comment:
Could you move this code into a dedicated module (Proxy::DHCP::Pingable perhaps?).

Reply:
Done